src/components/UserInfo.js

```diff
@@ -10,13 +11,12 @@ export default class UserInfo {
       job: this._jobElement.textContent,
     };
   }
-
   setUserInfo({ name, job }) {
     this._nameElement.textContent = name;
     this._jobElement.textContent = job;
   }
 
-  updateProfileInfo(newName, newJob) {
-    this.setUserInfo({ name: newName, job: newJob });
+  setAvatar(avatar) {
```

Comment:
It's great to have a separate setter for the avatar as it's controlled by a different form